lib/geoblacklight/wms_layer.rb

```diff
@@ -1,6 +1,7 @@
 module Geoblacklight
   class WmsLayer
     def initialize(params)
+      params = params.to_unsafe_h unless params.is_a?(Hash)
       @params = params.to_h.merge(Settings.WMS_PARAMS)
```

Comment:
doesn't the `params.to_h` already convert the params object into a Hash? maybe add a comment as to why `to_unsafe_h` is needed here